app/models/report.js

```diff
@@ -38,6 +38,16 @@ export default class Report extends Model {
    */
   @attr("django-duration", { defaultValue: () => moment.duration() }) duration;
 
+  /**
+   * The remaining effort for the underlying task
+   *
+   * @property {moment.duration} remainingEffort
+   * @public
+   */
```

Comment:
Are these strings still needed for something in timed? Maybe worth looking into since we could probably remove most of them.

Reply:
Asked this myself too. Maybe @anehx can tell us, since they come from the :sparkles: glorious :sparkles:  ages when he rigged that app.

Reply:
Those docstrings exist primarily because a part of Timed was my thesis where documentation was a part of the criteria. Drop it if the string doesn't have any added value.

Reply:
Thanks for the clarification! :+1: